content/docs/setup/kubernetes/install/multicluster/shared-gateways/index.md

```diff
@@ -1,19 +1,22 @@
 ---
-title: Cluster-Aware Service Routing
+title: Shared control plane (multi-network)
 description: Leveraging Istio's Split-horizon EDS to create a multicluster mesh.
+description: Install an Istio mesh across multiple Kubernetes clusters using a shared control plane for diconnected cluster networks.
 weight: 85
 keywords: [kubernetes,multicluster]
 aliases:
     - /docs/examples/multicluster/split-horizon-eds/
+    - /docs/tasks/multicluster/split-horizon-eds/
 ---
 
-This example shows how to configure a multicluster mesh with a
-[single control plane topology](/docs/concepts/multicluster-deployments/#single-control-plane-topology)
-and using Istio's _Split-horizon EDS (Endpoints Discovery Service)_ feature (introduced in Istio 1.1) to
+Follow this guide to configure a multicluster mesh with a
+[shared control plane topology](/docs/concepts/multicluster-deployments/#shared-control-plane-topology)
+and using Istio's _Split-horizon EDS (Endpoints Discovery Service)_ feature to
```

Comment:
Split horizon EDS is an implementation detail, I am not sure we should mention it at all. I do not think it will help the users in any way. We can mention it in the concept page, but I do not think that it is relevant in a task the users exercise.